src/iterative_wrappers.jl

```diff
@@ -17,6 +17,7 @@ end
 
 KrylovJL_CG(args...; kwargs...) = KrylovJL(args...; KrylovAlg = Krylov.cg!, kwargs...)
 KrylovJL_GMRES(args...; kwargs...) = KrylovJL(args...; KrylovAlg = Krylov.gmres!, kwargs...)
+KrylovJL_FGMRES(args...; kwargs...) = KrylovJL(args...; KrylovAlg = Krylov.fgmres!, kwargs...)
```

Comment:
add to the docs?

Reply:
I think that we don't need both `KrylovJL_GMRES` and `KrylovJL_FGMRES` after all.
I just added `KrylovJL_CRAIGMR` for least-norm problems and updated the documentation.